mycoffee/params.py

```diff
@@ -4,6 +4,8 @@
 
 MY_COFFEE_VERSION = "1.2"
 INPUT_ERROR_MESSAGE = "[Error] Wrong input"
+POSITIVE_INTEGER_ERROR_MESSAGE = "invalid positive-int value: '{string}'"
+POSITIVE_FLOAT_ERROR_MESSAGE = "invalid positive-float value: '{string}'"
```

Comment:
I think capitalizing the first character and making it `positive X` instead of `positive-X` is better:
```
POSITIVE_INTEGER_ERROR_MESSAGE = "Invalid positive int value: '{string}'"
POSITIVE_FLOAT_ERROR_MESSAGE = "Invalid positive float value: '{string}'"
```

Reply:
fixed in https://github.com/sepandhaghighi/mycoffee/pull/88/commits/146f8676e07464b163b460ad683605cda80ec46e and https://github.com/sepandhaghighi/mycoffee/pull/88/commits/ed3bfec59f28557f1af3e6528e41ea86aa4016aa